content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -135,6 +150,18 @@ You can check the status of the deployed Istio pods with:
 $ kubectl get pods -n istio-system
 {{< /text >}}
 
-## Uninstall
+## Uninstall the previous control plane
+
+If you have upgraded all dataplane components to use the new version of Istio, and are satisfied that you do not need to rollback, you can remove the previous version of the control plane by running:
+
+{{< text syntax=bash snip_id=show_istiod_values >}}
+$ helm delete istiod-"$REVISION" -n istio-system
+{{< /text >}}
+
+## Upgrade the CRDs
+
+Due to the difficulty of downgrading custom resource definitions, ({{< gloss >}}CRDs{{</ gloss >}}) should be upgraded only once you are confident you do not need to rollback to the previous version:
 
-Please refer to the uninstall section in the [Helm installation guide](/docs/ambient/install/helm-installation/#uninstall).
+    {{< text syntax=bash snip_id=manual_crd_upgrade >}}
+    $ kubectl apply -f manifests/charts/base/crds
```

Comment:
fwiw https://deploy-preview-15247--preliminary-istio.netlify.app/latest/docs/setup/upgrade/helm/#in-place-upgrade does it the other order. I don't have a strong preference one way or another, but it seems like they ought to be consistent

Reply:
Really it seems like most of the doc could be shared (or at least very similar) with just some additional steps for ambient

Reply:
Looking at it, putting it under `Operations` and linking to it as an "advanced upgrade strategy" makes sense, yeah.